sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -69,110 +93,224 @@ public static void init(@NotNull OptionsConfiguration<SentryOptions> optionsConf
     hub.close();
   }
 
+  /**
+   * Close the SDK
+   */
   public static synchronized void close() {
-    IHub hub = getCurrentHub();
+    IHub hub = getCurrentHub(); // TODO: should we not close all the hubs?
```

Comment:
@bruno-garcia

Reply:
This is a gap in the API. `Close` only closes the current Hub and this is the behavior in all SDKs AFAIK.
@HazAT was there any discussion over this issue?

We could hold weak references and dispatch close to all known hubs or something like that.
@mitsuhiko might have opinions here.

Reply:
I think it's fine just to close the current hub/client.
I would expect to call close doing it for all hubs.